package.json

```diff
@@ -7,6 +7,7 @@
     "persona-scripts": "dist/src/persona-scripts/index.js"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
in case anyone tries to run it on an older npm that doesn't trigger resolutions automatically